magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/check_coord_db.py

```diff
@@ -0,0 +1,156 @@
+    lst_key = config_dict["database_keys"]["LST"]
+    lst_df = pd.read_hdf(lst_h5, lst_key)
+    df = pd.read_hdf(
+        config_dict["database_paths"]["MAGIC+LST1"],
+        key=config_dict["database_keys"]["MAGIC+LST1"],
+    )  # TODO: put this file in a shared folder
+    df2 = pd.read_hdf(
+        config_dict["database_paths"]["MAGIC+LST1_bis"],
+        key=config_dict["database_keys"]["MAGIC+LST1_bis"],
+    )  # TODO: put this file in a shared folder
+    df = pd.concat([df, df2]).drop_duplicates(subset="LST1_run", keep="first")
+
+    df = df.drop_duplicates(subset=["source"])
+    df = df.reset_index()
+
+    date_lst = pd.to_datetime(df["DATE"], format="%Y%m%d")
+
+    delta = pd.Timedelta("1 day")
+    date_magic = date_lst + delta
+
+    date_magic = date_magic.dt.strftime("%Y/%m/%d").to_list()
+
+    for i in range(len(df)):
+        magic_first_run = (
+            (df["MAGIC_runs"].to_list())[i].rstrip("]").lstrip("[").split(", ")[0]
+        )
+        lst_run = (df["LST1_run"].to_list())[i]
+        source = (df["source"].to_list())[i]
+        if (
+            len(
+                glob.glob(
+                    f"/fefs/onsite/common/MAGIC/data/M[12]/event/Calibrated/{date_magic[i]}/*{magic_first_run}*{source}*.root"
+                )
+            )
+            == 0
+        ):
+            print(
+                f"No MAGIC file for {source}, {date_magic[i]} (MAGIC convention), run {magic_first_run}\n\n"
+            )
+            continue
+        magic_file = glob.glob(
+            f"/fefs/onsite/common/MAGIC/data/M[12]/event/Calibrated/{date_magic[i]}/*{magic_first_run}*{source}*.root"
+        )[0]
+
+        if len(lst_df.loc[lst_df["LST1_run"] == lst_run]) == 0:
+            print(f"{source}, run {lst_run}, not in LST database\n\n")
+            continue
+        ra_db = lst_df.loc[lst_df["LST1_run"] == lst_run]["ra"].values[0]
+        dec_db = lst_df.loc[lst_df["LST1_run"] == lst_run]["dec"].values[0]
+
+        if (np.isnan(ra_db)) or (np.isnan(dec_db)):
+            print(f"RA or Dec is NaN in the database for {source}\n\n")
+            continue
+
+        file_tree = uproot.open(magic_file)
+        branch_list = [
+            "MRawRunHeader.fSourceRA",
+            "MRawRunHeader.fSourceDEC",
+        ]
+        coord = file_tree["RunHeaders"].arrays(branch_list, library="np")
+        ra_file = (
+            coord["MRawRunHeader.fSourceRA"][0] / seconds_per_hour * degrees_per_hour
+        )
+        dec_file = coord["MRawRunHeader.fSourceDEC"][0] / seconds_per_hour
+
+        # check RF declination line
+        with open(args.dec_mc) as f:
+            dec_mc = np.asarray(json.load(f)).astype(np.float64)
+
+        mc_dec_file = float(dec_mc[np.argmin(np.abs(dec_file - dec_mc))])
+        mc_dec_db = lst_df.loc[lst_df["LST1_run"] == lst_run]["MC_dec"].values[0]
+        if (
+            angular_separation(lon1=ra_file, lat1=dec_file, lon2=ra_db, lat2=dec_db)
```

Comment:
angular_separation takes arguments in radians (or in units of angle), but I think you have here a float of degrees